src/qtgui/freqctrl.cpp

```diff
@@ -879,8 +879,10 @@ void CFreqCtrl::setFrequencyFocus()
     uint8_t position = floor(log10(m_freq));
     position = (uint8_t)fmax(position, 4);      // restrict min to 100s of kHz
 
+    const QPointF localPos = m_DigitInfo[position].dQRect.center();
     QMouseEvent mouseEvent(QEvent::MouseMove,
-                           m_DigitInfo[position].dQRect.center(),
+                           localPos,
+                           mapToGlobal(localPos),
```

Comment:
```
mapToGlobal(QPoint(localPos.x(),localPos.y())),
```
`mapToGlobal` expects `QPoint` instead of `QPointF` as an argument.

Reply:
Great the tests are here, I really did not paid attention to Qt5 compatibility.

Reply:
> @pinkavaj Let me know if you'd like to make these changes. If not, I can clean up `setFrequencyFocus` tonight or tomorrow.

I'm already on it, I have taken the liberty to change it a bit more, opinion welcome.

Reply:
That's outdated. You have pushed a better commit.